dev-util/mh/mh-1.0.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Generate help for Makefile"
+HOMEPAGE="https://github.com/oz123/mh"
+SRC_URI="https://github.com/oz123/mh/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+IUSE="test"
+KEYWORDS="~amd64"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-libs/libpcre2"
```

Comment:
so you're linking `libpcre2`, but not pulling it in a build time?

Reply:
My bad. Will be fixed.